src/seedu/addressbook/data/person/Printable.java

```diff
@@ -0,0 +1,9 @@
+package seedu.addressbook.data.person;
+
+/**
+ * A read-only immutable interface for a person data object.
+ * There should be a printable string representation of the object.
+ */
+public interface Printable {
+    String getPrintableString();
```

Comment:
Missing header comments for `getPrintableString`. According to our Java coding standard, `Write descriptive header comments for all public classes/methods.`